archunit-junit/junit5/api/src/main/java/com/tngtech/archunit/junit/conditions/Conditions.java

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright 2014-2022 TNG Technology Consulting GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.tngtech.archunit.junit.conditions;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import com.tngtech.archunit.PublicAPI;
+import com.tngtech.archunit.PublicAPI.State;
+import com.tngtech.archunit.PublicAPI.Usage;
+import com.tngtech.archunit.junit.internal.AdapterFor;
+import com.tngtech.archunit.junit.ArchTest;
+import org.junit.jupiter.api.condition.JRE;
+import org.junit.jupiter.api.condition.OS;
+
+
+/**
+ * A collection of wrappers for declaring {@code DisabledIfXxx} annotations on {@link ArchTest} annotated fields
+ */
+@Target({ })
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+public @interface Conditions {
+
+    @Target(ElementType.FIELD)
```

Comment:
Note that for method-flavored ArchUnit tests, the default `org.junit.jupiter.api.condition` annotations now work just fine